### PR TITLE
WebDriver: Add option to use Qt networking

### PR DESCRIPTION
**Pull request by tcl3**
Using Qt networking when running web platform tests improves performance significantly. The time to run the subset of tests we run on CI drops from 21.9 seconds to 8.2 seconds on my machine.

### CODE REVIEW

**Comment by ADKaster** [comment]
Did you want to add such a flag for WebWorker as well? Currently we'll always create a RequestServer so that workers can use it.

**Comment by tcl3** [comment]
> Did you want to add such a flag for WebWorker as well? Currently we'll always create a RequestServer so that workers can use it.

I've updated the PR to allow WebWorker to optionally use Qt networking. Running with Qt networking also seems to solve #189, although I'm not sure we can close it since there is still a hang when lagom networking is used.

File: Ladybird/WebWorker/CMakeLists.txt [hunk 1]
```diff
@@ -2,9 +2,7 @@ include(fontconfig)
 
 set(WEBWORKER_SOURCE_DIR ${LADYBIRD_SOURCE_DIR}/Userland/Services/WebWorker)
 
-set(CMAKE_AUTOMOC OFF)
-set(CMAKE_AUTORCC OFF)
-set(CMAKE_AUTOUIC OFF)
+set_target_properties(webcontent PROPERTIES AUTOMOC OFF AUTORCC OFF AUTOUIC OFF)
```
Comment (ADKaster) [inline Ladybird/WebWorker/CMakeLists.txt:5]: you'll need this on the webworker library, due to macOS using a case-insenstive filesystem 🙃 . copy pasta from WebContent?

Comment (tcl3) [reply Ladybird/WebWorker/CMakeLists.txt]: Yeah, this was totally a copy paste mistake :sweat_smile:. MacOS CI seems happy now.

**Pull request merged**